src/en/news/blog/2022/scaletesting-with-pawsey/index.md

```diff
@@ -0,0 +1,82 @@
+- A bug was identified in the scale up profile of the [`pg_autoscaler`](https://docs.ceph.com/en/quincy/rados/operations/placement-groups/#autoscaling-placement-groups), which prevented pg’s from peering. This has been subsequently fixed, and backported to [Pacific](https://github.com/ceph/ceph/pull/44869) and [Octopus](https://github.com/ceph/ceph/pull/44541).
+
+### Orchestrator/cephadm
+
+- `ceph-volume` was enhanced to better handle unexpected disk formats like Atari partitions and GPT based devices
+- `cephadm` uses ssh as the control path to each cluster host. During our negative testing, we found that when ssh issues were encountered (bad/missing keys), the resulting information returned to the user was overly verbose. This was fixed in [PR 45132](https://github.com/ceph/ceph/pull/45132).
+- Cephadm’s experimental 'agent' feature was used to provide the orchestrator with host, device and daemon state metadata. Testing showed that deployment and data quality were as expected, but the agent generated too much log traffic, making potential troubleshooting problematic. The intent is for the Agent to become a supported feature in the Ceph Reef release.
+- With so many hosts, the lack of filtering in the `orch host ls command` made interacting with the cluster more difficult. Filtering was added in [PR 44020](https://github.com/ceph/ceph/pull/44020), addressing this usability issue.
+
+### UI Management
+
+Historically, Ceph clusters of this size have been managed with a CLI-based toolchain, but we took this opportunity to briefly look at the integrated Ceph UI (dashboard).
+
+Whilst the monitoring capabilities of the dashboard were functional, management of hosts and OSDs were problematic leading to browser timeouts.
+
+The lack of pagination has been identified as the root cause for most of these responsiveness issues. Work is currently under way to introduce pagination to the dashboard's backend services layer ([PR 46644](https://github.com/ceph/ceph/pull/46644/)).
+
+### Monitoring
+
+Out-of-the-box, `cephadm` deploys a Prometheus server to provide the cluster with monitoring and alerting capabilities. The data sent to Prometheus is emitted by the mgr/prometheus module which interacts with ceph-mgr to present Ceph counters in Prometheus exposition format.
+
+As the cluster scaled beyond 2,800 OSDs the Prometheus server was unable to consistently retrieve data from the mgr/prometheus endpoint, primarily due to the volume of data returned (> 400K samples from a single endpoint).
+
+Even with the monitoring issues, we were able to determine some sizing guidelines for a Prometheus instance managing a 69PB Ceph cluster (YMMV!)
+
+- use an NVME or SSD device for the prometheus TSDB (3000-6000 IOPS peak demand)
+- if you have to share a mon host, use a separate device for the TSDB to avoid introducing latency to the ceph monitor store
```

Comment:
NVMe devices *are* SSDs.  Suggest:
"- Use SSDs, ideally NVMe, for the Prometheus TSDB, which can receive 3000-6000 peak IOPS. This may be a good role for QLC"

Capitalize If and Ceph

Reply:
capitalisation done. 
nvme are indeed ssd's but not all ssds are nvme's :)  I was using the distinction here since nvme = pcie based devices vs ssd which could be sata.
I have reworded.

I won't include QLC as a suggestion here since that's not what was tested, and given the large write and flush behaviour of prometheus, I'd probably opt for TLC over QLC anyway.